run.py

```diff
@@ -1,6 +1,14 @@
 from rh_ui.app_setup import create_app
+from rh_ui.security import build_response_headers
 
 app = create_app()
 
+
+@app.after_request
+def add_security_headers(resp):
+    resp.headers.extend(build_response_headers())
+    return resp
```

Comment:
This module isn't a good place to modify the app, methods of running it which call `create_app` directly instead of using this module would not include this hook.

To implement this in our case (with the `create_app` function pattern), we might need to use the blueprint hook instead, so these hooks are always applied at app creation.
https://flask.palletsprojects.com/en/2.3.x/api/#flask.Blueprint.after_app_request

Reply:
Also, why do we need our own post request hook when we're using the Talisman module? Is it to set extra headers that talisman doesn't support? Or am I misunderstanding the purpose/function of Talisman? Either way, if we need both, it would be good to add a few doc strings explaining why.